draft-ietf-rats-corim.md

```diff
@@ -456,9 +487,37 @@ The following describes each child item of this group.
 ~~~
 
 
-# CoMID {#sec-comid}
+# Concise Module Identifier (CoMID) {#sec-comid}
+
+A CoMID tag contains information about hardware, firmware, or module composition that is provisioned
+in a Verifier. Each CoMID has a unique identifier (tag-id) that is used to unambigously identify
+tag instances. The tag-id may be used to cross reference CoMID tags from other manifest files (CoRIMs)
+and also within the same manifest, such as CoBOM, CoSWID, or TA-Store.
+
+A CoMID defines several types of Claims, using triples. A triple is a statement linking one object
+(subject) to another object (object) or a literal, via a predicate.
+
+Each triple defines a specific type of Claim, as given below.
+
+* Reference Triples contain Claims for Reference Values that are expected to match the attested Evidence.
+
```

Comment:
kramdown-rfc itemisation does not require empty lines between items, only before the first item.
```suggestion
```